frontend/packages/component-library/src/section/Section.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import {HTMLAttributes, ReactNode} from 'react';
+
+import moduleStyles from './section.module.scss';
+
+export interface SectionProps extends HTMLAttributes<HTMLElement> {
+  /** Background color */
+  backgroundColor?:
+    | 'primary'
+    | 'secondary'
+    | 'dark'
+    | 'brand-light-primary'
+    | 'brand-light-secondary';
+  /** Vertical padding */
+  padding?: 'm' | 'l';
+  /** Section content */
+  children?: ReactNode;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/Section.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ### Status: ```Ready for dev```
+ *
+ * Design System: Section Component.
+ * Acts as a container for section content in the Contentful CMS.
+ */
+const Section: React.FC<SectionProps> = ({
+  backgroundColor = 'primary',
+  padding = 'l',
+  children,
+  className,
+  ...HTMLAttributes
+}: SectionProps) => {
+  return (
+    <section
+      className={classNames(
+        moduleStyles.section,
+        moduleStyles[`section-${backgroundColor}`],
+        moduleStyles[`section-${padding}`],
+        className,
+      )}
+      data-testid="section"
```

Comment:
In apps folder linter we have a linter rule that deprecates the use of `data-testid` attribute and encourages to test components in the way that the client will see it (meaning test content instead for example)

I wonder if there's a way to avoid using `data-testid` here

(will left some comment with proposal in the test file)